src/app/inventory/DraggableInventoryItem.tsx

```diff
@@ -19,6 +19,9 @@ export default function DraggableInventoryItem({ children, item }: Props) {
       ? item.equipment
       : item.equipment || item.bucket.hasTransferDestination;
 
+  const inPostmasterCantTransfer =
+    item.location.inPostmaster && (!item.canPullFromPostmaster || item.notransfer);
```

Comment:
`notransfer` means "can't be transferred between characters/vault". It's probably not what you want/need.

Reply:
Removing the `item.notransfer` seems to show that I can still pull these items even when I don't have inventory space
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/54964587/210660137-ee7b38be-3f3b-402f-af63-ec7705e90579.png">

As opposed to leaving it in
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/54964587/210660194-34bc6487-f552-4a42-abb0-c28f92bc7174.png">

Reply:
But it also means they'd be greyed out even when you do have the space :) 
Here's how other code determines this dynamic aspect https://github.com/DestinyItemManager/DIM/blob/6c2831c54d233ef0b4f0dc3f93fcb23c85a1101c/src/app/item-popup/item-popup-actions.ts#L72-L80

Reply:
Awesome I'll take a look 😃